package.json

```diff
@@ -37,7 +37,7 @@
         "@typescript-eslint/parser": "^7.2.0",
         "@vencord/types": "^0.1.2",
         "dotenv": "^16.4.5",
-        "electron": "^29.1.1",
+        "electron": "29.1.6",
```

Comment:
?

Reply:
i didnt even notice i did that lol, was probably a remnant from testing. Only index.ts and ScreenSharePicker.tsx are necessary as far as I know